### DISCUSSION TITLE
Debug console window opens in release mode

### DISCUSSION THREAD

**Discussion by konst-sh** [AvaloniaUI/Avalonia#9930] [Q&A] [answered]
Hi, I'm new to desktop app development, testing Avalonia FuncUI project from the standard counter template and facing that debug console window opens when I'm running exe file. The project is compiled in Release mode. Any thoughts how to get rid of it? Thanks.

**Comment by danielmayost** [comment, marked as answer]
I think in `csproj` file you need:
```
<OutputType>WinExe</OutputType>
```

**Reply by konst-sh** [reply]
Thanks, it worked!
